templates/dotnet-publish-image.yaml

```diff
@@ -0,0 +1,43 @@
+{{- $env_vars := list
+}}
+{{- $parameters := list
+    "PROJECT"
+    "IMAGE_NAME"
+    "USE_DOTNET_IMAGESTREAM_BASE_IMAGES"
+    "VERBOSITY"
+    "BUILD_PROPS"
+}}
+{{- $wrkspaces := list
```

Comment:
Is this a type ? `wrkspaces` ?

Reply:
Ah it's indented, any reason to name it that way ? (it's an absolute nit of a comment 😛 )

Reply:
I am assuming it should work fine… but now that you said it, I have a doubt 😂

Reply:
I changed it, and it continues to work.